User Guide/Projects.md

```diff
@@ -121,40 +129,49 @@ After creating a new project, go to **Connections** tab. This tab shows the **co
 - SSL Client certificate
 - SSL Private key 
 - Private key password - you will receive this in your email.
+- Click Download certificate, pop-up to download the certs in .zip, enter a password to save the file in your local.
+- Alternatively you can also copy and save the certificates in a .PEM text editor file.
 
+**Note:**
+ *New **SSL certificate expires** after 2 years instead of 1 year*.
+ 
 ![Image not Available](/assets/Fig36.png)
 
- **Note:** *The **Private key password** for the projects, you will receive in your email. You must copy and save the certificates in a .PEM text editor file.*
+![Image not Available](/assets/Fig105_2.png)
+
+ **Note:** 
+ *The **Private key password** for the projects, you will receive in their email inbox*. 
```

Comment:
"the projects, you will receive in their email inbox" -> "the project will be sent to your email"

Reply:
done